src/term_rst_post/newspost.py

```diff
@@ -209,15 +209,13 @@ def depart_literal(self, node):
 
     def visit_reference(self, node):
         """ Encapsulate links with names """
-        if not self.beyond_limit:
-            if 'name' in node.attributes:
-                self.body.append('[')
+        pass
```

Comment:
just remove it?

Reply:
I prefer to keep all the `visit_*` and `depart_*` couples present in the code to clearly see what is being done to each node element in the doc.